examples/06_assets/main.rs

```diff
@@ -1,66 +1,32 @@
 extern crate amethyst;
 extern crate cgmath;
-extern crate obj;
 
-use amethyst::engine::{Application, State, Trans};
-use amethyst::context::{ContextConfig, Context};
 use amethyst::config::Element;
-use amethyst::ecs::{World, Join};
-use amethyst::context::asset_manager::{Assets, AssetLoader, AssetLoaderRaw, DirectoryStore, Mesh, Texture};
-use amethyst::renderer::VertexPosNormal;
-use cgmath::{InnerSpace, Vector3};
-use std::io::BufReader;
-
-struct Obj(obj::Obj);
```

Comment:
I'm wondering if there's any rationale behind keeping this here. Loading `*.obj` files seems like a a pretty common use case, so it made sense to me to move this directly to the asset manager module, but it could certainly be moved back to here.